cmd/clusterctl/client/cluster/client.go

```diff
@@ -242,13 +242,13 @@ func retryWithExponentialBackoff(ctx context.Context, opts wait.Backoff, operati
 
 // newWriteBackoff creates a new API Machinery backoff parameter set suitable for use with clusterctl write operations.
 func newWriteBackoff() wait.Backoff {
-	// Return a exponential backoff configuration which returns durations for a total time of ~40s.
-	// Example: 0, .5s, 1.2s, 2.3s, 4s, 6s, 10s, 16s, 24s, 37s
+	// Return a exponential backoff configuration which returns durations for a total time of ~6m.
+	// Example: 0, .5s, 1.2s, 2.3s, 4s, 6s, 10s, 16s, 24s, 37s, 56s, 84s, 126s, 190s
 	// Jitter is added as a random fraction of the duration multiplied by the jitter factor.
 	return wait.Backoff{
 		Duration: 500 * time.Millisecond,
 		Factor:   1.5,
-		Steps:    10,
+		Steps:    14,
```

Comment:
I wonder if we should tweak Factor to make the max interval size smaller? Perhaps increasing duration and status, but reducing factor could lead to a longer overall timeout but retain better UX with less excessive waiting?

Reply:
I though about that too but avoided for now to reduce the risk of introducing another race somewhere else.